includes/wc-payment-api/class-wc-payments-api-client.php

```diff
@@ -338,15 +338,21 @@ public function update_intention(
 	 *
 	 * @param string $intention_id - The ID of the intention to update.
 	 * @param array  $metadata     - Meta data values to be sent along with payment intent creation.
+	 * @param bolean $remove_receipt_email - Whether to remove the receipt_email.
 	 *
 	 * @return WC_Payments_API_Intention
 	 * @throws API_Exception - Exception thrown on intention creation failure.
 	 */
-	public function update_intention_metadata( $intention_id, $metadata ) {
+	public function update_intention_metadata( $intention_id, $metadata, $remove_receipt_email = false ) {
 		$request = [
 			'metadata' => $metadata,
 		];
 
+		if ( $remove_receipt_email ) {
+			// I tried to set this to null but then Stripe did not remove the email.
+			$request['receipt_email'] = '';
```

Comment:
Can we apply `$request['metadata']['receipt_email']` here and add a comment that it's for BC with older Mobile apps? That would allow us to omit other changes.